annotation_tool/README.md

```diff
@@ -27,4 +27,7 @@ The credentials should match in database image and application configuration.
 2. Run docker-compose by executing `docker-compose up`.
 
 
-3. The UI should be available at `localhost:7001`.
\ No newline at end of file
+3. The UI should be available at `localhost:7001`.
+
+# Usage
+The manual can be found [here](https://drive.google.com/file/d/1Wv3OIC0Z7ibHIzOm9Xw_r0gjTFmpl-33/view).
```

Comment:
This is a good addition. Unfortunately the annotation handbook is rather out of date. E.g. it does not include CSV type uploading. How about we explicitly mention the manual is outdated?

Reply:
@elyase I added a note, regarding the slightly outdated version of this handbook.